configs/a64fx_SME.yaml

```diff
@@ -294,13 +266,130 @@ Latencies:
     Execution-Latency: 6
     Execution-Throughput: 1
   11:
+    Instruction-Groups:
+    - ~1922,
```

Comment:
Similar to my comment in `a64fx.yaml` a brief comment here would be good

Reply:
Added a brief comment